tests/fixtures/napari.py

```diff
@@ -49,4 +49,42 @@ def _valid_dataset_with_localised_nans(nan_location):
 
         return (out_path, valid_poses_dataset)
 
-    return _valid_dataset_with_localised_nans
+    return _valid_poses_dataset_with_localised_nans
+
+
+@pytest.fixture
+def valid_poses_dataset_long(valid_poses_dataset, tmp_path):
+    """Return a (path, dataset) pair representing a poses dataset
+    with 2 individuals ("id_0" and "id_1") and 3 keypoints
+    ("centroid", "left", "right") moving in uniform linear
+    motion for 10 frames in 2D space.
```

Comment:
The only crucial thing to say here is that this fixture is based on the `valid_poses_dataset` fixture (the details about individuals and keypoints can be looked up in the "parent" fixure).

Perhpa mention the 10 frames though, is it pertains to the "long" bit.